_config.yml

```diff
@@ -1,91 +1,73 @@
-title: 	NEON Raster Data in R 
-description: 	A set of lessons .
-#Comment out url when working locally to resolve base urls correctly ok
-#use this when pushing to git for live version
+#------------------------------------------------------------
+# Values for this lesson.
+#------------------------------------------------------------
 
-url:      http://datacarpentry.org/NEON-R-Spatial-Raster
-baseurl:   http://datacarpentry.org/NEON-R-Spatial-Raster
+# Which carpentry is this ("swc", "dc", or "lc")?
+carpentry: "dc"
 
-#use this when working locally
-#just uncomment the first localhost.
-#url: http://localhost:4000
-#baseurl: http://localhost:4000
+# Overall title for pages.
+title: "Geospatial analysis with R"
```

Comment:
I think we might consider a different name given we aren't doing any analysis really... ie no real stats. More
Intro to geospatial data with R or Work with geospatial data in R